src/main/java/org/opensearch/ad/task/ADBatchTaskRunner.java

```diff
@@ -688,17 +680,19 @@ private synchronized void startNewEntityTaskLane(ADTask adTask, TransportService
     }
 
     private void dispatchTask(ADTask adTask, ActionListener<DiscoveryNode> listener) {
+        listener.onResponse(clusterService.localNode());
+        /* @anomaly.detection commented until we have support for the hashring, not necessary to query for node stats as all tasks are dispatches to extension node 
```

Comment:
This makes sense to comment this code out as it's basically to dispatch the tasks to different node having AD running and currently we are focusing on single node AD Extension. Can you link the hashring issue here?

Reply:
sure